scripts/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+
+# Install
+set -euo pipefail
+
+# RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_CUDA_SUFFIX=12
+# TODO: set this to main once dask-cudf is compatible
```

Comment:
I think https://github.com/rapidsai/dask-cuda/pull/1424 is the PR to track here. For now, using 2024.12.1 is slightly preferable, otherwise all the tests fail on import.